.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: Release Please
+
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**.md'
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    permissions:
+        id-token: write
+        contents: write
+        pull-requests: write
+    steps:
+      - name: Release Please  
+        uses: google-github-actions/release-please-action@v3
+        with:
+          token: ${{ secrets.RA_GH_RELEASE_TOKEN  }}
+          release-type: simple
+          package-name: actions
+          changelog-types: '[{"type":"feat","section":"Features","hidden":false},{"type":"fix","section":"Bug Fixes","hidden":false},{"type":"chore","section":"Miscellaneous","hidden":false},{"type":"refactor","section":"Refactor","hidden":false}]'
```

Comment:
Hey just a small note to add a new line here

Reply:
and on the other places this red sign shows up

Reply:
Will do !

Reply:
@o-leolleo all newline errors now resolved